src/applications/gi/sass/gi.scss

```diff
@@ -128,10 +128,6 @@
   }
 }
 
-.form-expanding-group-open {
```

Comment:
removing as only place needing this change to this class is the "Type of institution" filter on the Landing Page